config/config.yml

```diff
@@ -263,9 +263,8 @@ FIELDS_ALLOWING_LESS_THAN: ["query", "words", "kudos", "hits", "date", "bookmark
 #
 # This will ensure that the field has been sanitized with the latest version of the sanitizer.
 #
-FIELDS_ALLOWING_HTML: ["about_me", "comment", "content", "description", "endnotes", "faq", "intro", "note", "notes",
-  "rules", "signup_instructions_general", "signup_instructions_offers", "signup_instructions_requests", "summary",
-  "screencast", "disabled_support_form_text"]
+FIELDS_ALLOWING_HTML: ["about_me", "bookmark_notes", "comment", "comment_content", "content", "description", "disabled_support_form_text", "endnotes", "faq", "intro", "note", "notes",
```

Comment:
Should this be `bookmarker_notes`? (I'm not sure I follow the `bookmarker_notes` vs. `bookmark_notes` distinction yet, so feel free to ignore me if this is a silly question.)

Reply:
It should! I forgot to change it after my last minute change to the column name. I will fix that and add a test.